activerecord/test/cases/connection_adapters/connection_handler_test.rb

```diff
@@ -353,28 +353,6 @@ def test_forked_child_recovers_from_disconnected_parent
           end
         end
 
-        def test_retrieve_connection_pool_preserves_connection_specification_name
```

Comment:
I don't think this test makes much sense anymore as we're trying to eliminate the "pool name".